packages/material-ui/src/ClickAwayListener/ClickAwayListener.js

```diff
@@ -101,15 +116,23 @@ ClickAwayListener.propTypes = {
   /**
    * The mouse event to listen to. You can disable the listener by providing `false`.
    */
-  mouseEvent: PropTypes.oneOf(['onClick', 'onMouseDown', 'onMouseUp', false]),
+  mouseEvent: PropTypes.oneOf([
+    'click',
+    'onClick',
+    'mousedown',
```

Comment:
Is this overload important? Seems like we can save some branching by not supporting it. Seems like we didn't before anyway.

Reply:
I believe it's an effort toward eventually only supporting the standard event names: https://developer.mozilla.org/en-US/docs/Web/Events.

Reply:
Sounds fair though I would have preferred a separate PR so that it appears properly in a feature note in the changelog. We probably want to deprecate the `on` prefix later in feature and remove them in a breaking release.

Reply:
Yes, I think that it's a v5 effort.

Reply:
I'll split the migration out.